user_tools/src/spark_rapids_pytools/common/sys_storage.py

```diff
@@ -102,6 +102,17 @@ def copy_resource(cls, src: str, dest: str) -> str:
                                    f'Resource {abs_src} does not exist')
         return shutil.copy2(abs_src, abs_dest)
 
+    @classmethod
+    def copy_resources(cls, src_dir: str, dest_dir: str) -> str:
+        abs_src = os.path.abspath(src_dir)
+        abs_dest = os.path.abspath(dest_dir)
+
+        # check if path exists
+        if not os.path.exists(abs_src):
+            raise StorageException('Error copying resources on local disk. '
+                                   f'Resource {abs_src} does not exist')
+        return shutil.copytree(abs_src, abs_dest, dirs_exist_ok=True)
+
```

Comment:
Very similar to `copy_resource` (naming and code). 
You may want to either:

- combine the two into one by checking whether src is a file/directory
- change the name of the method to be easily distinguish from `copy_resource`. For example, `copy_dir_recursively`

Reply:
Combined the functions